src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(0.5),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chip: {
+      borderRadius: '4px',
+      color: 'white',
+      backgroundColor: ({ value }) => {
+        if (value === 'High') {
+          return secondary.main;
+        }
+
+        if (value === 'Medium') {
+          return warning.main;
+        }
+
+        return '#8f928b';
+      },
+    },
+  })
+);
+
+export interface SensitivityProps {
+  value: SensitivityType;
+}
+
+export const Sensitivity: FC<SensitivityProps> = (props) => {
+  const { icon, iconWrapper, chip } = useStyles(props);
+
+  return (
+    <Box>
+      <Box className={iconWrapper}>
+        <VerifiedUserOutlined className={icon} color="disabled" />
+        <Typography variant="body2" color="textSecondary">
```

Comment:
The color of this should be `#3C444E`. What's the best way to add that in to the theme config? `textSecondary` is a bit too light 🤷

Reply:
Based on the designs it looks like it's the primary text color.
That's defined here https://github.com/SeedCompany/cord-field/blob/d6e89115d4f8cf37cb75a69e47d273a9a1cbd0e8/src/theme/index.ts#L38
The color probably just changed in designs since I last looked.
I believe that's the default color for `Typography`, but don't take my word for it.